wimm.Secundatives.UnitTests/Extensions/Expect_Test.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using wimm.Secundatives.Exceptions;
+using wimm.Secundatives.Extensions;
+using Xunit;
+
+namespace wimm.Secundatives.UnitTests.Extensions
+{
+    public class Expect_Test
+    {
+        private const string _valid = "doot";
+        const string _message = "this is a message";
```

Comment:
private const?

Reply:
wf

Reply:
:shipit:

Reply:
(That is the shipit) markdown emoji apparently